agent/utils/uuidwrapper/uuid.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package uuidwrapper
```

Comment:
nit: rename this package to be the same as whatever you call the interface.

Reply:
i was mimicking other wrapper packages when i created this.. now that i don't use mock i think i will put it under utils package because it's pretty simple